docs/chain/collator.md

```diff
@@ -574,21 +547,22 @@ docker run -p 127.0.0.1:9933:9933 -v ~/data:/data \
 </TabItem>
 </Tabs>
 
-## Known Issues
+:::info
 
-The following are known issues and the possible solutions:
+The same process that includes generating and registering sessions keys and joining the collator candidate pool must be re-executed for Spiritnet.
 
-- running "cargo update -p environmental" could fix something
-- Using the commands create a `/Data/` directory the folder could possibly be owned by the root user. Switching the ownership of the folder with `chown [options] user:group filename`. Solution found [here](https://serverfault.com/questions/106713/how-to-change-ownership-of-a-folder-using-terminal)
+:::
 
 ## Troubleshooting
 
-There are a few things that you can check to make sure everything is setup correctly.
+There are a few things that can checked to make sure everything is setup correctly.
+
+If, from any network explorer, e.g., the one offered by PolkadotJS Apps, the collator's account is shown next to some of the blocks, then the collator is correctly producing blocks and getting rewarded for it.
+If the logs print the message that starts with a :gift: emoji it indicates that the collator setup is correct but that the blocks produced are not included by the relaychain. This typically signals some issues about the node hardware or connectivity.
```

Comment:
This is only true if the first step failed. That's the reason i added numbers. It's like if n-1 failed check n.

```suggestion
If the logs print the message that starts with a :gift: emoji it indicates that the collator setup is correct but that the blocks produced are not included by the relaychain. This typically signals some issues about the node hardware or connectivity.
```

Reply:
So after which point you would include this?

Reply:
i think the order is fine. it wasn't clear for me that this only matters when the first point was false.
It sounds like having the 🎁 in logs is something bad. Like it means that the block wasn't submitted.